Mysql/views/district_summary.sql

```diff
@@ -0,0 +1,24 @@
+ 
+
+SELECT 
+       `clusters`.`smd_id` AS `district_id`,
+       clusters.region_id as region_id,
```

Comment:
Should be `clusters`.`region_id` as `region_id`, the quotation marks are missing

Reply:
Ha, nicely caught. Not a syntax error (` quotes are only essential when using strings with dashes or ones involving sql keywords), but I should be consistent here... updating...